endpoints/openrtb2/amp_auction.go

```diff
@@ -843,9 +861,21 @@ func setTrace(req *openrtb2.BidRequest, value string) error {
 	return nil
 }
 
-// setSeatNonBid populates bidresponse.ext.prebid.seatnonbid if bidrequest.ext.prebid.returnallbidstatus is true
-func setSeatNonBid(finalExtBidResponse *openrtb_ext.ExtBidResponse, request *openrtb_ext.RequestWrapper, auctionResponse *exchange.AuctionResponse) bool {
-	if finalExtBidResponse == nil || auctionResponse == nil || request == nil {
+// setSeatNonBid populates bidresponse.ext.prebid.seatnonbid
+func setSeatNonBid(finalExtBidResponse *openrtb_ext.ExtBidResponse, seatNonBid []openrtb_ext.SeatNonBid) bool {
```

Comment:
Can we move this function inside auction.go? file as this is a common function for all types of auctions.

Reply:
yes, done